lib/stack.js

```diff
@@ -4,12 +4,16 @@
 
 impress.shortenStack = stack => {
   if (!stack) return '';
-  const nmPath = api.path.join(impress.dir, 'node_modules');
-  const libPath = api.path.join(impress.dir, 'lib/');
-  const result = stack
-    .replace(api.common.newEscapedRegExp(nmPath), '')
-    .replace(api.common.newEscapedRegExp(libPath), '')
-    .replace(api.common.newEscapedRegExp(impress.dir), '');
+  if (!stack.includes(impress.dir)) stack;
+  const nmPath = api.path.sep + 'node_modules';
+  const libPath = api.path.sep + 'lib' + api.path.sep;
```

Comment:
I think we don't need `api.path.sep` because we don't support beauty optimizations for windows and node.js supports both `/` and `\` in paths, even mixed.